### PR TITLE
Fix/admin icons

### PR DESCRIPTION
**Pull request by Camwyn**
Correct issue where admin sidebar icons disappear.
Also did some fixes for the linter - there are about 60 duplicated selectors in there - didn't touch those ☹️ 

Bad: 
bad

Good: https://d.pr/i/S9173e

Ref: https://theeventscalendar.atlassian.net/browse/TEC-3741

### CODE REVIEW

**Review by bordoni — DISMISSED** [review]

**Comment by paulmskim** [comment]
@Camwyn I believe I actually fixed most of these in my linting pass for tribe common, let's merge that in and see if your fixes still apply.

File: src/resources/postcss/tribe-common-admin.pcss
Comment (paulmskim) [inline src/resources/postcss/tribe-common-admin.pcss:1603]: With the previous order, the `border-color` will overwrite the `border-bottom-color`. So in the fixed version, we should really only need `border-color` and remove the `border-bottom-color`, as it doesn't take effect.

File: src/resources/postcss/tribe-common-admin.pcss
Comment (paulmskim) [inline src/resources/postcss/tribe-common-admin.pcss:3701]: maybe make this something like:

```
.tribe-events-admin__system-information-widget-copy button {

	.dashicons,
	.dashicons-before:before {
		...
	}
}
```

Comment (paulmskim) [inline src/resources/postcss/tribe-common-admin.pcss:3701]: maybe make this something like:

```
.tribe-events-admin__system-information-widget-copy button {

	.dashicons,
	.dashicons-before:before {
		...
	}
}
```

**Review by paulmskim — APPROVED** [review]

**Comment by paulmskim** [comment]
@Camwyn I'd hold off on merging this until we get the linting stuff merged forward, there might be merge conflicts and it'd be nice to see what conflicts there are before merging. But otherwise, it's good to go.

**Comment by EugeneKyale** [comment]
Hey @paulmskim is it safe to merge this now? I've resolved some conflicts that were related to some updates I made to this file recently.

File: src/resources/postcss/tribe-common-admin.pcss
Comment (paulmskim) [inline src/resources/postcss/tribe-common-admin.pcss:3986]:
```suggestion
	@media only screen and (max-width: 480px) {
```

File: src/resources/postcss/tribe-common-admin.pcss
Comment (paulmskim) [inline src/resources/postcss/tribe-common-admin.pcss:4030]:
```suggestion
	@media only screen and (max-width: 480px) {
```

File: src/resources/postcss/tribe-common-admin.pcss
Comment (paulmskim) [inline src/resources/postcss/tribe-common-admin.pcss:4048]:
```suggestion
	@media only screen and (max-width: 480px) {
```

**Review by paulmskim — COMMENTED** [review]
a few spacing fixes

**Comment by paulmskim** [comment]
@EugeneKyale I don't think danish is merged forward into empanada yet, we should hold off until that is done.

**Comment by EugeneKyale** [comment]
@paulmskim The issue is that the icons are missing in release/B21.empanada which will go out before danish.

**Comment by EugeneKyale** [comment]
Thanks @paulmskim I've updated the spacing issues.

**Review by paulmskim — APPROVED** [review]

**Pull request merged**
